Makefile

```diff
@@ -0,0 +1,36 @@
+Command := $(firstword $(MAKECMDGOALS))
+Arguments := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+
+default: show 
+
+bhyveImage:
+	@echo "nix building $(Arguments) bhyve Image"
+	nix build ".#nixosConfigurations.$(Arguments).config.system.build.bhyveImage"
+
+qemuVm:
+	@echo "nix building $(Arguments) qemu VM"
+	nix build ".#nixosConfigurations.$(Arguments).config.system.build.vm"
+
+all:	bhyveImage qemuvm
```

Comment:
This should be `qemuVm`

Also, if we built both result will be overriden in the cwd which Im ok with but maybe its worth adding a warning to the user since they might expect `result`, `result2`, etc.

Reply:
oh yeah, good catch, i can probably change the results symlink name to some different, but didn't because nixxers tend to build one things an expect it to be place in result.

Reply:
Im fine with keeping to the convention of result and if I want something I can run a subsequent command to get `result` linked to the store.

Reply:
I've put them into different symlink outputs, so there should be no confusion now.